internal/distro/fedora31/distro.go

```diff
@@ -366,48 +364,55 @@ func New() *Fedora31 {
 			"tar",
 			"xz",
 		},
-		arches: map[string]fedora31Arch{
-			"x86_64": {
-				name: "x86_64",
-				bootloaderPackages: []string{
-					"grub2-pc",
-				},
-				buildPackages: []string{
-					"grub2-pc",
-				},
-				imageTypes: map[string]imageType{
-					"ami": amiImgType,
-					"ext4-filesystem": ext4FilesystemType,
-					"partitioned-disk": partitionedDisk,
-					"qcow2": qcow2ImageType,
-					"openstack": openstackImgType,
-					"tar": tarImgType,
-					"vhd": vhdImgType,
-					"vmdk": vmdkImgType,
-				},
-			},
-			"aarch64": {
-				name: "aarch64",
-				bootloaderPackages: []string{
-					"dracut-config-generic",
-					"efibootmgr",
-					"grub2-efi-aa64",
-					"grub2-tools",
-					"shim-aa64",
-				},
-				uefi: true,
-				imageTypes: map[string]imageType{
-					"ami": amiImgType,
-					"ext4-filesystem": ext4FilesystemType,
-					"partitioned-disk": partitionedDisk,
-					"qcow2": qcow2ImageType,
-					"openstack": openstackImgType,
-					"tar": tarImgType,
-					"vhd": vhdImgType,
-				},
-			},
+		arches: map[string]fedora31Arch{},
+	}
+
+	x8664 := fedora31Arch{
```

Comment:
Not `x86_64`?

Reply:
Camel case linter is complaining about the underscore.

Reply:
Ah, fair enough.